docs/blockchainwallets.md

```diff
@@ -0,0 +1,95 @@
+# Blockchain Wallets
+
+# What is a crypto wallet?
+
+In essence, a cryptocurrency wallet is a gadget that allows you to communicate with a blockchain network. There are various types of cryptocurrency wallets, which can be divided into three categories: software, hardware, and paper wallets. Depending on how they work, they may alternatively be referred to as "hot" or "cold" wallets.

```

Comment:
The word gadget feels weird here. I've never heard of an application called as a gadget.